docs/data-sources/env_azure_status.md

```diff
@@ -51,4 +51,4 @@ Read-Only:
 
 Read-Only:
 
-- `private_link_service_alias` (String) VPC endpoint service name in $endpoint_service_id.$region.vpce.amazonaws.com format (if any)
+- `private_link_service_alias` (String) Globally unique name for your service in prefix.GUID.suffix format..
```

Comment:
I suggest we change this to `Private Link Service Alias / DNS Name`. `... in prefix.GUID.suffix format` is really confusing.